content/post/session3-startupdeepdive.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Start-Up Deep Dive : Revenue Model and Building Sustainable Products from Kerala"
+date: 2023-01-25T00:55:13+05:30
+Description: "A highlight of the third session of Startup Deep Dive taken by Felix Josemon on the topic “Revenue Model”."
+authors:
+  - jesa-dileep-philip
+Tags: [ "startup","startupdeepdive","revenue-models"]
+Categories: ["startup"]
+DisableComments: false
+draft: true
+---
+**STARTUP DEEP DIVE** is an Entrepreneurship Training Series conducted by IEDC MEC which aims at providing individuals with the aptitude to identify business opportunities, self-esteem, knowledge, and skills to act on them. Entrepreneurs or the move towards self-employment is and will continue to become, an increasingly important element of economic growth and development.
+
+<br>
+<img src="/images/session3-startupdeepdive/img1.png" width="550" height="auto">
+<br>
+
+>**"Start-ups don't fail because they lack a product, but they fail because they lack customers and a profitable business model"** <br>
+> <p style="text-align: right"> ~ Steve Blank ( American Entrepreneur ) </p>
```

Comment:
What about just this?✨

> "Start-ups don't fail because they lack a product, but they fail because they lack customers and a profitable business model"
> ~ Steve Blank ( American Entrepreneur )

Reply:
left align the author name?

Reply:
yep